Server/Game/routegamehelper.py

```diff
@@ -200,6 +204,11 @@ def queryGamesUser(data):
 
 	"""
 	username = data['session_username']
+	filters = {}
+	if 'Filters' in data:
+		filters = data['Filters']
+		logging.info("QGU Filters: {}".format(filters))
```

Comment:
Does this need to be info level? Should probably be debug.

Reply:
This, in my opinion, should either stay at info, since I think it is appropriate as an info type message, or it should be taken out completely.  We already log every request that comes in so this message is probably unnecessary in general.  I think I'll take it out.